CHANGELOG.md

```diff
@@ -4,9 +4,12 @@ All notable changes to this project will be documented in this file.
 The format is based on [Keep a Changelog](https://keepachangelog.com/en/1.0.0/),
 and this project adheres to [Semantic Versioning](https://semver.org/spec/v2.0.0.html).
 
-### Fixed
+### Changed
 - Remove redundant `'assert'` dependency from the code. (#672)
 
+### Fixed
+- Fixed library support for IE11. The `unorm` package is added to the dependencies (#675)
+
 ## [Unreleased]
```

Comment:
Something went wrong here, `## [Unreleased]` should be above the changelog entries

Reply:
Fixed within https://github.com/handsontable/hyperformula/pull/675/commits/65ea318fd047eefc22852ef50bc64072f5c11830